<?php
namespace Home\Controller;

use Think\Controller;
use Think\Exception;

class AssociationController extends BaseController
{
    public function index()
    {
        $this->title = '社群';
        $this->display();
    }
     public function demo()
    {
        $this->display();
    }
    public function associationList()
    {
        $_GET['cate_id'] ? $_GET['cate_id'] : 0;
        try {
            if($_GET['cate_id'] == 0){
                $list = D('Association')->getList(array('status' => 1));
                foreach ($list as $k => $v) {
                    $list[$k]['logo'] = M('Organization')->where(array('token' => $v['token']))->getField('picture');
                }
            }else{
                $list = D('Association')->getList(array('status' => 1, 'cate_id' => $_GET['cate_id']));
                foreach ($list as $k => $v) {
                    $list[$k]['logo'] = M('Organization')->where(array('token' => $v['token']))->getField('picture');
                }
            }
            $cateList = D('ProductClass')->getList();
            $data = [
                'list' => $list,
                'cateList' => $cateList
            ];
            return show(1, '', $data);
            // $(this)->display();
        } catch (Exception $e) {
            return show(0, $e->getMessage());
        }
    }
    public function expertDetails()
    {
        $this->title = '专家详情';
        $this->display();
    }
}